cmake/Embed.cmake

```diff
@@ -97,23 +95,23 @@ function(generate_embed_source EMBED_NAME)
         string(REGEX REPLACE ".[A-Za-z0-9_]+$" "" BASE_NAME ${BASE_NAME})
 
         string(APPEND INIT_KERNELS "
-            { \"${BASE_NAME}\", { ${START_SYMBOL}, ${END_SYMBOL}} },
+            { \"migraphx/kernels/${BASE_NAME}\", { ${START_SYMBOL}, ${LENGTH_SYMBOL} } },
```

Comment:
Do not prepend `migraphx/kernels/` to the name. Not all the files we plan to embed in the future will be in the same path.

Reply:
Noted, I will revert that.

Reply:
> However, how would you plan to distinguish between files that need to be prepended from those that are not required to be prepended?

For kernel files we will probably use the `RELATIVE` parameter like what we do in CK. For other types of files, we will probably make a completely different embed library to use.